spec/integration_spec.rb

```diff
@@ -358,6 +358,7 @@
     # Apple products
     @iphone = Product.create!(name: 'iphone', href: 'apple', tags: ['awesome', 'poor reception'],
                               description: 'Puts even more features at your fingertips')
+    @macbook = Product.create!(name: 'macbookpro', href: 'apple')
```

Comment:
I didn't get why you had to create a new product here 😬. Can you explain it to me?

Reply:
In my previous code, the `synonyms` setting also converted it to CamelCase if it contained `_`.
So I added a product to test cases where synonyms contain `_`.

And
https://github.com/meilisearch/meilisearch-rails/blob/a7eace0e97ae02413c2ec53ac5e789c472f9df76/spec/integration_spec.rb#L500-L502
↑upThis test case is
https://github.com/meilisearch/meilisearch-rails/blob/a7eace0e97ae02413c2ec53ac5e789c472f9df76/spec/integration_spec.rb#L445
where by iphone has been removed.
The test was not as intended because the iphone has been removed here, effectively making it `expect(0).to eq(0)`.
So I added a new product to the existing test case with the impact on the content.